core/backend/CHANGELOG.md

```diff
@@ -692,7 +702,7 @@ Mon, 26 Jul 2021 12:21:25 GMT
 
 - Reactivated test that stopped working when a previous PR for ecef dependency to geographicCRS was reverted
 - store changesetIndex on IModelDb
-- Better error checks when creating SpatialViewDefinition-s.
+- Better error checks when creating SpatialViewDefinition-s. 
```

Comment:
spacing issues again...

Reply:
Since all the mds were regenerated from the changelog jsons (based off of Dan's script), this spacing change actually fixes the issue brought up previously. We shouldn't be seeing this moving forward